.ci/updatecli/updatecli-bump-stack-version.yml

```diff
@@ -0,0 +1,51 @@
+# update-cli configuration for automated go updates
+---
+name: Bump ESS snapshots
+
+pipelineid: 'bump-ess-snapshot-{{ requiredEnv "BRANCH" }}'
+
+scms:
+  githubConfig:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GITHUB_ACTOR" }}'
+      username: '{{ requiredEnv "GITHUB_ACTOR" }}'
+      owner: "{{ .scm.owner }}"
+      repository: "{{ .scm.repository }}"
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      branch: '{{ requiredEnv "BRANCH_NAME" }}'
+      commitusingapi: true
+      force: false
+
+sources:
+  lastStableStackSnapshot:
+    name: "Latest Stable ESS snapshot for current branch"
+    kind: json
+    spec:
+      file: 'https://storage.googleapis.com/artifacts-api/channels/{{ requiredEnv "BRANCH_NAME" }}.json'
+      key: .build
+targets:
+  updateStackImages:
+    name: "Update create_deployment_csp_configuration.yaml"
+    sourceid: lastStableStackSnapshot
+    scmid: githubConfig
+    kind: file
+    spec:
+      file: "pkg/testing/ess/create_deployment_csp_configuration.yaml"
+      matchpattern: '(\S*_image: ".*docker.*):(.+)"'
+      replacepattern: '$1:{{ source "lastStableStackSnapshot" }}-SNAPSHOT"'
+actions:
+  elastic-agent:
+    kind: github/pullrequest
+    scmid: githubConfig
+    sourceid: lastStableStackSnapshot
+    spec:
+      automerge: true
```

Comment:
is the automerge enabled with this PR or is it gonna be a follow-up? I see this issue https://github.com/elastic/observability-robots/issues/2853 and I am not sure I follow